css/index.css

```diff
@@ -59,8 +61,53 @@ footer {
     background-color: #E5C185;
     padding: 20px;
 }
-@media all {
-        body {
-            background-color: #FBF2C4;
-        }  
+#Message {
+    display: grid;
+    grid-template-columns: max-content;
+    grid-template-rows: max-content;
+}
+a {
+    display: grid;
+    color: #E5C185;
+    grid-template-rows: 100%;
+    grid-template-columns: 100%;
+}
+/*design for smartphones*/
+@media screen and (width: minmax( 320px, 480px)){
+    .navbar li {
+        display: grid;
+        background-color: #C7522A;
+        justify-content: space-between;
+        padding: relative;
+        padding-left: 20%;
+        position: sticky;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: auto;
+    }
+}
+/*design for tablets*/
+@media screen and (width: minmax( 480px, 768px)){
+    .navbar li {
+        display: grid;
+        background-color: #C7522A;
+        justify-content: space-between;
+        padding: relative;
+        padding-left: 20%;
+        position: sticky;
+        grid-template-columns: 1fr 1fr 1fr;
+        grid-template-rows: auto;
+    }
+}
+/*design for desktop*/
+@media screen and (width: minmax( 768px, 1200px)){
+    .navbar li {
+        display: grid;
+        background-color: #C7522A;
+        justify-content: space-between;
+        padding-left: 20%;
+        padding: 20px;
```

Comment:
... and this one is almost the same. It's usually good to remember DRY: "Don't repeat yourself" ... If you mean for two things to be the same, they should share a definition.

Reply:
Since there are effectively just two ranges, maybe just use min-width or max-width?